view_post.php

```diff
@@ -0,0 +1,217 @@
+				"NOTLI: UK",
+				"Childline: <span>0800 1111",
+				"Samaritans: <span>08457 909090",
+				"Abuse Not: <span>0808 8005015",
+				"Brook Young People's Information Service: <span>0800 0185023",
+				"Eating Disorder Support: <span>01494 793223",
+				"Anxiety UK: <span>0844 477 5774",
+				"Depression Alliance: <span>0845 123 23 20",
+				"Rape Crisis Centre: <span>01708 765200",
+				"Rape/sexual assault: <span>0808 8000 123 (female) or 0808 8000122 (male)",
+				"Miscarriage Association: <span>01924 200799",
+				"LLGS Helpline (LGBT): <span>0300 330 0630",
+				"Sexuality support: <span>01708 765200",
+				"Bereavment: <span>0800 9177 416",
+				"Runaway/homeless: <span>0808 800 70 70",
+				"CareConfidential Pregnancy/post abortion: <span>0800 028 2228",
+				"Women’s Aid National",
+				"Domestic Violence Helpline <span>0345 023 468",
+				"National AIDS Helpline: <span>0800 567 123</li></span>"
+			),
+			'header2'=>'',
+			'header3'=>'',
+			'sugg'=>false,
+			'ol'=>false,
+			'ul'=>true,
+			'links'=>false
+		),
+		'8'=>array(
+			'title'=>'alternatives',
+			'header1'=>'alternatives to self harm',
+			'p'=>'',
+			'listelement'=> array("draw lines or doodles where you want to cut",
+									"play some loud music<br>
+									(suggested playlists: <a target='_blank'href='https://open.spotify.com/user/spotify/playlist/37i9dQZF1DX4fpCWaHOned?si=YbaxeAjaTcGqWhClPaMMkA'>Spotify playlist</a>)"
+			),
+			'header2'=>'alternatives:',
+			'header3'=>'personal suggestions',
+			'sugg'=>true,
+			'ol'=>true,
+			'ul'=>false,
+			'links'=>false
+		),
+		'9'=>array(
+			'title'=>'playlists',
+			'header1'=>'playlists',
+			'p'=>'a compilation of playlists made to fit particular moods or themes',
+			'listelement'=> '',
+			'header2'=>'',
+			'header3'=>'',
+			'linkarray'=>array(
+				'tired/quiet moods'=>'https://open.spotify.com/user/22k343m6upq3jz5sylnsoshsa/playlist/2qPlEDx72NGKkKhJkgIw1A?si=I7-UH57XRrqBuQiHqQ7x9g',
+				'light/warm moods'=>'https://open.spotify.com/user/22k343m6upq3jz5sylnsoshsa/playlist/6RH5vWE1Cvd3bItS13KrCd?si=nmtixRLCTWOkTHtntjS_Lw',
+				'cold/icy moods'=>'https://open.spotify.com/user/22k343m6upq3jz5sylnsoshsa/playlist/6npCX62HU1NYi804GFzWCW?si=QlES2yAiQfeK7rM6Yyrc7w',
+				'heavy edm'=>'https://open.spotify.com/user/22k343m6upq3jz5sylnsoshsa/playlist/6IKXyKgX75A5p67ZbQqvvt?si=GOO6dpPEQaKH__uacKBXEA',
+				'monthly'=>'https://open.spotify.com/user/22k343m6upq3jz5sylnsoshsa/playlist/6jd9clieyQgb3UyKZaMaQc?si=S5ZS9j8tQCiY0oXwM-3COQ',
+				'summer pop'=>'https://open.spotify.com/user/22k343m6upq3jz5sylnsoshsa/playlist/1yQIUrswAZhbIpzL9vLd4b?si=FOp19-FTSheioNaDfki58Q',
+				'classic rock and others from the same era'=>'https://open.spotify.com/user/22k343m6upq3jz5sylnsoshsa/playlist/6yalt8yoor2dfaBpG8qYWQ?si=eJWaJKSbRSaFwCMY6meKfg',
+			),
+			'sugg'=>false,
+			'ol'=>false,
+			'ul'=>false,
+			'links'=>true
+		)
+	);
+
+	echo"
+		<html>
+			<head>
+				<title>".
+				$posts[$_REQUEST['postID']]['title']."</title>
```

Comment:
Since you're calling `$posts[$_REQUEST['postID']]` a bunch of places here, I'd suggest saving a separate variable like this:
```
$post = $posts[$_REQUEST['postID']];
```
Then you could just reference `$post` so it'd clean your code up a bit.